src/components/Calculator.css

```diff
@@ -39,14 +43,16 @@ span {
 
 .btn-opr,
 .btn-opr-right {
-  width: 25%;
+  width: 50%;
   border: 1px solid rgb(147, 145, 145);
+  height: 20vh;
```

Comment:
Just as the previous reviewer asked, your calculator height spans the viewport height and does not look great as the template given and it also causes scrolling. But I have made some changes in a few areas you should modify your code to get things right 👍

- [ ] Kindly have a height set to around 50px `height: 50px;`. You can replace the height you had earlier 👍